nixos/modules/services/networking/cloudflared.nix

```diff
@@ -217,8 +214,7 @@ in
                 };
 
                 path = mkOption {
-                  type = with types; nullOr str;
-                  default = null;
```

Comment:
With these changes I'm getting:
```
error: The option `services.cloudflared.tunnels.xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx.ingress."hostname".path' is used but not defined.
```
Why are all these default values being removed?

Reply:
Oops, I didn't realise this was for the submodule - in my configuration I'm just using `attrsOf str`. I've reverted this.